.github/workflows/release.yaml

```diff
@@ -6,11 +6,13 @@ on:
       - "v*"
 
 permissions:
-  contents: write
+  contents: read
```

Comment:
this will break the next release, the contents: write is required by two other jobs in this workflow